### ISSUE TITLE
Exception in thread "Thread-25" java.security.AccessControlException: access denied (java.lang.RuntimePermission modifyThread)

### ISSUE THREAD

**Issue by epishkin** [MasseGuillaume/ScalaKata#1]
macosx 10.7.4
sbt 0.11.3
scala 2.9.1
java 1.6.0_33

when I run code from http://scala.masgui.cloudbees.net/50088c7fe4b0cfd655485a58 on my laptop in ScalaKata (copy&paste code from that URL into ScalaKata running on my laptop) I get this exception

```
Exception in thread "Thread-33" java.security.AccessControlException: access denied (java.lang.RuntimePermission modifyThread)
    at java.security.AccessControlContext.checkPermission(AccessControlContext.java:374)
    at java.security.AccessController.checkPermission(AccessController.java:546)
    at java.lang.SecurityManager.checkPermission(SecurityManager.java:532)
    at scala.actors.threadpool.ThreadPoolExecutor.checkShutdownAccess(Unknown Source)
    at scala.actors.threadpool.ThreadPoolExecutor.shutdown(Unknown Source)
    at scala.actors.scheduler.ResizableThreadPoolScheduler.run(ResizableThreadPoolScheduler.scala:144)
```

**Comment by MasseGuillaume** [comment]
resolved in ec67fcb115ad804584fa215330ae5ee7d8e0f67c

**Comment by epishkin** [comment]
thanks!
